site/en/blog/iframe-credentialless/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: 'layouts/blog-post.njk'
+title: "Iframe credentialless: Easily embed iframes in COEP environments"
+description: >
+  Iframe credentialless is implemented in Chrome 110.  It provides developers a way to load documents in third-party iframes using a new and ephemeral context. In return, they are no longer subject to the COEP embedding rules. 
+  Developers using COEP can now embed third party iframes that do not use COEP themselves.
+subhead: >
+  Developers using COEP can now embed third party iframes that do not use COEP themselves.
+date: 2023-01-12
+authors:
+  - arthursonzogni
+hero: image/udVScdcCFAdRjZwFdLk2jWAFQyr1/unmhfaD1hmGZfa8DF04f.jpg
+alt: A photo of a person sitting on a bench.  
+tags:
+  - privacy
+  - security
+---
+
+Iframe credentialless is enabled by default from Chrome version 110. It solves the most common complaint developers working with [Cross-Origin-Embedder-Policy (COEP)](https://web.dev/security-headers/#coep) have: embedding third-party iframes that do not set COEP.
+
+{% Aside %}  
+It was previously available as an origin trial from version 106 to 108, and known as [anonymous iframe](/blog/anonymous-iframe-origin-trial/).
+{% endAside %}
+
+## Why we need COEP
+
+Some web APIs increase the risk of side-channel attacks such as [Spectre](https://en.wikipedia.org/wiki/Spectre_(security_vulnerability)). To mitigate that risk, browsers offer an opt-in-based isolated environment called [cross-origin isolation](https://web.dev/coop-coep/), which requires deploying COEP. [Cross-origin isolation](https://web.dev/coop-coep/) allows websites to use privileged features including [`SharedArrayBuffer`](/blog/enabling-shared-array-buffer/), [`performance.measureUserAgentSpecificMemory()`](https://web.dev/monitor-total-page-memory-usage/), and [high-precision timers with better resolution](/blog/cross-origin-isolated-hr-timers/).
+
+To enable cross-origin isolation, websites must send the following HTTP headers:  
+`Cross-Origin-Embedder-Policy: require-corp`  
+`Cross-Origin-Opener-Policy: same-origin`
```

Comment:
Could we add syntax color for http?
```text
|  ```http
|  Cross-Origin-Embedder-Policy: require-corp
|  Cross-Origin-Opener-Policy: same-origin
|  ```
```
=>
```http
Cross-Origin-Embedder-Policy: require-corp
Cross-Origin-Opener-Policy: same-origin
```

Reply:
done